jobs/build/coreos-installer_sync/Jenkinsfile

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env groovy
+node {
+    checkout scm
+    def buildlib = load("pipeline-scripts/buildlib.groovy")
+    def commonlib = buildlib.commonlib
+
+    commonlib.describeJob("coreos-installer_sync", """
+        <h2>Sync contents of the coreos-installer RPM to mirror</h2>
+        http://mirror.openshift.com/pub/openshift-v4/clients/coreos-installer/
+
+        Timing: This is only ever run by humans, upon request.
+    """)
+
+    properties([
+        disableResume(),
+        buildDiscarder(
+            logRotator(
+                artifactDaysToKeepStr: '',
+                artifactNumToKeepStr: '',
+                daysToKeepStr: '',
+                numToKeepStr: ''
+            )
+        ),
+        [
+            $class: 'ParametersDefinitionProperty',
+            parameterDefinitions: [
+                string(
+                    name: "NVR",
+                    description: "NVR of the blessed brew build",
+                    defaultValue: "",
+                    trim: true,
+                ),
+                string(
+                    name: "VERSION",
+                    description: "Desired version name. Example: v3.0.1",
+                    defaultValue: "",
+                    trim: true,
+                ),
+                string(
+                    name: "ARCHES",
+                    description: "Arches to extract from the brew build. Defaults to all contained in the rpm",
+                    defaultValue: "",
+                    trim: true,
+                ),
+                commonlib.dryrunParam(),
+                commonlib.mockParam(),
+            ],
+        ]
+    ])
+
+    workdir = "${env.WORKSPACE}/coreos-installer_sync"
+    commonlib.checkMock()
+    buildlib.initialize()
+    buildlib.cleanWorkdir(workdir)
+
+    stage("Validate params") {
+        if (!params.NVR) {
+            error "NVR must be specified"
+        }
+        if (!params.VERSION) {
+            error "VERSION must be specified"
+        }
+    }
+
+    stage("Download RPM") {
+        commonlib.shell(
+            script: [
+                "set -exuo pipefail",
+                "cd ${workdir}",
+                "rm --recursive --force ./*",
+                "brew --quiet download-build ${params.NVR}",
+                "rm *bootinfra* *.src.rpm",
+                "tree",
+            ].join('\n'),
```

Comment:
can I suggest using a `"""`-delimited string for these embedded scripts? cuts down on punctuation to trip you up.

```
           script: """
                set -exuo pipefail
                cd ${workdir}
                rm --recursive --force ./*
                brew --quiet download-build ${params.NVR}
                rm *bootinfra* *.src.rpm
                tree
                """
```